.github/workflows/test-turbo.yml

```diff
@@ -113,7 +113,7 @@ jobs:
             - name: Run tests
               working-directory: src/Turbo
               env:
-                  SYMFONY_DEPRECATIONS_HELPER: max[direct]=0
+                  SYMFONY_DEPRECATIONS_HELPER: max[direct]=0&baselineFile=./Tests/allowed.json
```

Comment:
This is a workaround for some deprecations, which should be fixed by https://github.com/symfony/symfony/pull/43241

Reply:
I guess this is now merged, could we remove it?